### DISCUSSION TITLE
any way to add custom styles? (am sponsor, like hosted map)

### DISCUSSION THREAD

**Discussion by seanvosler** [tone-row/flowchart-fun#318] [Q&A] [answered]
Loving the flowchart tool so far! BIG FAN... cant see what you do with it and Roam Research.

Curious, is there a way to style a chart in the header?

Would also likely need the ability to add a class to a given type (could be done with #hastags?)

For example...
~~~
theme: museum
style:
    .people{background:blue;}
   .food{background:yellow;}
~~~
John Doe #person (gets class person)
    Is a human
    likes pancakes #food (gets class food)

**Comment by seanvosler** [comment]
Ah i see how it works, in the documentation it's mapped out (couldn't find docs in github but found on site) If anyone's looking in the future here are the details: https://flowchart.fun/h#classes

But still cant figure out how to attach a style to a given class... thoughts?

**Comment by rob-gordon** [comment, marked as answer]
@seanvosler Thank you for sponsoring! 🙏 

Here's an example of adding styles to classes: https://flowchart.fun/c#H4hQGcBcE8BsFMBcoAEKC0LzwQY0gPYBOiKARAHQC206usAhuOGamljAsu+wEYO4A1gHMiBAK4A7ACZ0CsYqWg4FAdzY9c8xSl6xx8DewBmBSZHSr4AS2EALSKV7zpRtKfPpjDKtdjRSXHEicGsAN0MQYFAAbWpaeiZwAF0UABU7a3AUBgAHXJRVYkFs3mgUGFzrSWE2AEkZeHNq4RRcIngGSHhshhRYasEKggq7eDbgjvN+6sM0Bkly7oAPR114Uw6ctu1JNo6unu3GXhw2AGED7uPBlGlrDvx-FHFQmtHx+GWBSH6GU9gFS+kCMA0E40gmXApAAFBksjl8oViqUltAqjUAJRGGJBKAEKgoOoAEXidEYzFSxDcunKDGk93eCxQMRJqQW0hQHWM8CmuBaoy6NLQYIhUNheMIhJJmJQAHo5SgAJoSNrMhiwcAjV7jN7CBDoAaScZaKhUJqQcCgOUAKlA1Ko4lgkD8s1ApvN5itoAAEgwIihjFIAIQoQA8G4BI-dANrlQA

Let me know if you have any other questions!

**Reply by seanvosler** [reply]
AHH! thank you... i knew it had to be something like this, but couldnt quite get the syntax right. Thanks again!

``` for future reference for anyone who needs....

style:
  - selector: ".people"
    style:
      background-color: yellow
      color: blue
      font-weight: bold
```
